galsim/chromatic.py

```diff
@@ -367,6 +746,8 @@ def copy(self):
 
     # Helper function
     def _applyMatrix(self, J):
+        if hasattr(J, '__call__'):
+            self.noInterpolation()
```

Comment:
Doesn't this call mean that the check in `_intep_drawImage` won't ever trigger?  If so, I think this is where the warning should be emitted.

Might also need a similar thing when applying a chromatic flux scaling.